### ISSUE TITLE
[Vulkan] Crash on startup with Radeon 7770

### ISSUE THREAD

**Issue by AsureDawn** [libretro/RetroArch#3245]
When vulkan driver is enabled, RetroArch crashes immediately upon opening.
Cmd output (-v):[log.txt](https://github.com/libretro/RetroArch/files/371685/log.txt)
Hardware/software info: [system_info.txt](https://github.com/libretro/RetroArch/files/371686/system_info.txt)
Debug dump file: [retroarch_debug.exe.8660.dmp](https://drive.google.com/open?id=0B6HZjBTsAO7EUktCbld1ZHFYM3M)

**Comment by AsureDawn** [comment]
Changing the menu driver to rgui as mentioned in Issue #3229 doesn't help--still crashes immediately.
Tried in my r9 270x rig, still crashes immediately (with both xmb and rgui drivers).

**Comment by AsureDawn** [comment]
Removed all AMD drivers with amdcleanuputility, reinstalled driver (16.3.2), reran with vulkan driver and rgui: crashes at same place.

Ran using SDL2 driver, loaded paraLLEl core and ran NUTS; got about as far as the vulkan driver did before crashing:
[SDL2_paraLLEl_NUTS.txt](https://github.com/libretro/RetroArch/files/380096/SDL2_paraLLEl_NUTS.txt)

**Comment by AsureDawn** [comment]
Replaced r9 270x with RX 480, works now (both the xmb/rgui with vulkan and paraLLEl, though it did crash when I got to a certain point in paper mario, in addition to a few graphical glitches). The 480 needed a newer driver than the r9 270x, so it may have been an incompatibility with the older driver.

I still have the 270x if you want to pursue a solution to this, it's just not hooked up right now.
